README.md

```diff
@@ -319,39 +338,41 @@ Contributions of any kind welcome!
 
 MIT
 
-[prettier]: https://github.com/jlongster/prettier
-[eslint]: http://eslint.org/
-[fix]: http://eslint.org/docs/user-guide/command-line-interface#fix
-[npm]: https://www.npmjs.com/
-[node]: https://nodejs.org
+[all-contributors]: https://github.com/kentcdodds/all-contributors
```

Comment:
Not sure why you changed the order of the links

Reply:
Alphabetized. No real reason for it. Just felt awkward slapping the first-defined links at the end of the list.